.github/workflows/sonarcloud.yml

```
@@ -1,7 +1,11 @@
 ---
 name: sonarcloud
 
-on: [push]
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)